ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/workspace/WorkspaceServiceClient.java

```diff
@@ -116,10 +116,14 @@
      *         workspace ID
      * @param envName
      *         the name of the workspace environment that should be used for start
+     * @param isRecover
```

Comment:
What about option when server decides recover or not depending on auto-restore behavior?